web/src/main/webapp/js/instructions.js

```diff
@@ -91,6 +89,27 @@ module.exports.create = function (mapLayer, path, urlForHistory, request) {
         instructionsElement.append(moreDiv);
     }
 
+    if (debugDetails) {
+        var detailObj = path.details;
+        for (var detailKey in detailObj) {
+            var pathDetailObj = detailObj[detailKey]["details"];
+            for (var k in pathDetailObj) {
+                var intervalArr = pathDetailObj[k];
+                for (var intervalIndex in intervalArr) {
+                    var interval = intervalArr[intervalIndex];
+                    var lngLat = path.points.coordinates[interval[0]];
+                    L.marker([lngLat[1], lngLat[0]], {
+                        icon: L.icon({
+                            iconUrl: './img/marker-small-blue.png',
+                            iconSize: [15, 15]
+                        }),
+                        draggable: true
+                    }).addTo(mapLayer.getRoutingLayer()).bindPopup(detailKey+":"+k);
+                }
+            }
+        }
+    }
```

Comment:
Ok. Thought about this. The response is fine, maybe we add later a GeoJSON compatible conversion as possibility. But what I would like to change is the way we request the details, it should not require 2 parameters and instead we should either have a semicolon-based list like `details=average_speed;something_else` like we allow for `blocking_area` or just `details=average_speed&details=something_else` like e.g. for the point array.